assets/js/add-another.js

```diff
@@ -3,8 +3,9 @@
  */
 var AddAnother = {}
 
-AddAnother = function(container) {
+AddAnother = function(container, removeContainerSelector) {
   this.container = $(container)
+  this.removeContainerSelector = removeContainerSelector
   this.container.on('click', '.add-another__remove-button', $.proxy(this, 'onRemoveButtonClick'))
   this.container.on('click', '.add-another__add-button', $.proxy(this, 'onAddButtonClick'))
   this.container.find('.add-another__add-button, add-another__remove-button').prop('type', 'button')
```

Comment:
Why is this required?

Reply:
Previously, the add another component would clone the element and then append a remove button to it. This provides more control so we can choose where exactly the remove button is placed